tests/FINDOLOGIC/Export/Tests/DataHelperTest.php

```diff
@@ -100,4 +101,19 @@ public function numericValueProvider()
             'zero as string' => ['0', false]
         ];
     }
+
+    /**
+     * @expectedException \FINDOLOGIC\Export\Helpers\ValueExceedsInternalCharacterLimitException
+     */
+    public function testValueNotExceedingCharacterLimit()
```

Comment:
Let the name of the test show that this refers to the length of attribute values.

Reply:
It would also be nice to have a unit test for `DataHelper::checkValueNotExceedingCharacterLimit` because it could generally called from anywhere, and it would no longer be covered if it were no longer be called by attributes.

Reply:
Done :heavy_check_mark: